Code/GraphMol/MarvinParse/testMrvToMol.cpp

```diff
@@ -866,6 +866,39 @@ class MrvTests {
     BOOST_LOG(rdInfoLog) << "done" << std::endl;
   }
 
+  void testMarvinRgroupError() {
+    BOOST_LOG(rdInfoLog) << "testing marvin Rgroup error" << std::endl;
+    std::string rdbase = getenv("RDBASE");
+    const auto molb = R"CTAB( 
+  -INDIGO-06252516172D
+
+  2  1  0  0  0  0  0  0  0  0999 V2000
+    9.9900   -7.1250    0.0000 R#  0  0  0  0  0  0  0  0  0  0  0  0
+    8.9900   -7.1250    0.0000 O   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  1  0  0  0  0
+M  RGP  1   1   1
+M  END
+    )CTAB";
+    std::unique_ptr<ROMol> mol = v2::FileParsers::MolFromMolBlock(molb);
+
+    SmilesWriteParams ps;
+    ps.canonical = false;
+
+    std::string smilesOut = MolToSmiles(*mol, ps);
+    std::string cxSmilesOut = MolToCXSmiles(*mol, ps);
+
+    auto outMolStr = MolToMrvBlock(*mol, true, -1, true, false);
+
+    std::unique_ptr<ROMol> mol2 =
```

Comment:
Please also add tests for the explicit change that you made, i.e. that the isotope and (as suggested above) `common_properties::_MolFileRLabel` property are set.

Reply:
done